plugins/newrelic-dashboard/README.md

```diff
@@ -0,0 +1,79 @@
+# New Relic Dashboard Plugin
+
+Welcome to the new-relic-dashboard plugin!
+
+## Features
+
+- Adds New Relic Dashboard Pages Links to Overview section of the catalog
+- Shows Snapshots of dashboards in New Relic
+
+## Getting started
+
+This plugin uses the Backstage proxy to securely communicate with New Relic's APIs. We use NerdGraph (New Relic's GraphQL API)
+
+To generate a New Relic API Key , you can visit this [link](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)
+
+1. Add the following to your app-config.yaml to enable this configuration:
+
+```
+// app-config.yaml
+proxy:
+  '/newrelic/apm/api':
+    target: https://api.newrelic.com/v2
+    headers:
+      X-Api-Key: ${NEW_RELIC_USER_KEY}
+```
+
+2. Add the following to `EntityPage.tsx` to display New Relic Dashboard Tab
+
+```
+// In packages/app/src/components/catalog/EntityPage.tsx
+import {
+  isNewRelicDashboardAvailable,
+  EntityNewRelicDashboard,
+  EntityPageNewRelicDashboard,
+} from '@internal/plugin-new-relic-dashboard';
```

Comment:
need to fix this to be `@backstage/plugin-newrelic-dashboard` i think?

Reply:
Oh , yes . Forgot to change the readme . (There was a renaming which had happened) . Thanks for checking that. @adamdmharvey